### ISSUE TITLE
Creature complexity setting

### ISSUE THREAD

**Issue by ninapaley** [toolness/mystic-symbolic#18]
Set complexity between 1 and, say, 10 (although complexity hypothetically can go up to infinity).

0 = just a symbol by itself
1 = one set of attachments or nests
2 = 2 sets of attachments, or 1 set of attachments + 1 set of nests
3 = 3 sets of attachments, or 2 sets of attachments + 1 nest, or 1 set of attachments + 2 nests
etc. 

Examples:
complexity = 0:
![complexity_0](https://user-images.githubusercontent.com/79212762/108601963-153ad380-7365-11eb-9033-0857692cda70.png)
complexity = 1 (many ways for this to go):
![complexity1a](https://user-images.githubusercontent.com/79212762/108601977-326fa200-7365-11eb-96c3-b3cd3e60dd8c.png)
![complexity1b](https://user-images.githubusercontent.com/79212762/108601979-36032900-7365-11eb-8331-41b7e12b0a15.png)
![complexity1c](https://user-images.githubusercontent.com/79212762/108601981-38658300-7365-11eb-878b-d74c159c4df6.png)
![complexity1d](https://user-images.githubusercontent.com/79212762/108601983-3a2f4680-7365-11eb-9f93-03c352a63787.png)
![complexity1e](https://user-images.githubusercontent.com/79212762/108601986-3c91a080-7365-11eb-9db1-cfdf53d7c38c.png)

complexity = 2 (also many possibilities, I'm only including a few to illustrate):
![complexity2a](https://user-images.githubusercontent.com/79212762/108602005-503d0700-7365-11eb-9c6c-978a9072e7ce.png)
![complexity2b](https://user-images.githubusercontent.com/79212762/108602006-57641500-7365-11eb-9d59-c0b1ca50958a.png)
![complexity2c](https://user-images.githubusercontent.com/79212762/108602009-59c66f00-7365-11eb-8be6-f36cabb46753.png)
![complexity2d](https://user-images.githubusercontent.com/79212762/108602011-5b903280-7365-11eb-8123-bccec432ac23.png)

complexity = 3:
![complexity3a](https://user-images.githubusercontent.com/79212762/108602018-65199a80-7365-11eb-9c60-21f37d29319e.png)
![complexity3b](https://user-images.githubusercontent.com/79212762/108602020-677bf480-7365-11eb-9cb4-a83c3bff0cb1.png)

complexity = 4:
![complexity4a](https://user-images.githubusercontent.com/79212762/108602024-6f3b9900-7365-11eb-91ca-9ee36608c180.png)
![complexity4b](https://user-images.githubusercontent.com/79212762/108602028-71055c80-7365-11eb-9645-56bff47db057.png)

complexity = 5:
![complexity5](https://user-images.githubusercontent.com/79212762/108602031-7793d400-7365-11eb-9ba6-dbd0fc4cf57b.png)

**Comment by ninapaley** [comment]
I guess what we have is a matrix of potential symbols, and complexity sets how many symbols in this matrix are rendered. See animated gif below. The higher the level (shown here as smaller polygons), the more possible configurations. Since I made this manually I gave up before finishing every possibility at level 2.

I don't really care if the complexity setting is "accurate," I just want a way to make simpler images without crowding the canvas with stuff, as well as more complex images like what we have. If there is a way for fake or approximate a complexity setting, that's fine with me too.
![complexity matrix_1](https://user-images.githubusercontent.com/79212762/108728696-71316380-74ef-11eb-9b9e-320940106e56.gif)
